config/config.go

```diff
@@ -308,6 +309,8 @@ type Config struct {
 	MuteTimeIntervals []MuteTimeInterval `yaml:"mute_time_intervals,omitempty" json:"mute_time_intervals,omitempty"`
 	TimeIntervals     []TimeInterval     `yaml:"time_intervals,omitempty" json:"time_intervals,omitempty"`
 
+	TracingConfig TracingConfig `yaml:"tracing,omitempty" json:"tracing,omitempty"`
```

Comment:
That was my first instinct actually, but in `prometheus/prometheus` it's not under `global`. Should we prefer to keep things under `global`, or follow the pattern established in `prometheus/prometheus`?

Reply:
Hmm. It looks like the Prometheus configuration file is structured a little different from Alertmanager as the Prometheus configuration file also has options to change the behavior of the Prometheus server, rather than just contain data like `rules`. However, Alertmanager configuration just contains data, and all behavioral configuration is instead done via command line flags. @gotjosh what should we do here?

Reply:
I read on prometheus-users that if the configuration is expected to be reloadable at runtime it should go in the configuration file, otherwise it should go in command line args.

https://groups.google.com/g/prometheus-users/c/qYKpBiuN8y4/m/9z062yU4CQAJ